src/frontends/tensorflow/src/op/avg_pool.cpp

```diff
@@ -21,23 +21,29 @@ OutputVector translate_avg_pool_op(const NodeContext& node) {
     auto tf_padding_type = node.get_attribute<std::string>("padding");
     auto tf_data_format = node.get_attribute<std::string>("data_format");
 
```

Comment:
Ideally we should check lenghts for strides, kernel size to avoid out-of-memory access. I know that TF specification says but we need to guard ourselves because we don't know of data received from OVTF as an example.